### ISSUE TITLE
Regex Validation Not working Properly ..Error is throw opposite 

### ISSUE THREAD

**Issue by ayyaNaga** [running-coder/jquery-form-validation#8]
Hi 
I am using URL Regex in my code
While validate the form empty filed is working fine but I Given the Correct web address "http://www.google.com" it will throw the error once again given the wrong Web address ""htsdftp://www.google.com"" This will accept so how to solve the problem.

data-validation="[NOTEMPTY]" 
data-validation-message="Link cannot be empty" 
data-validation-regex="/^(https?:\/\/)?((([a-z0-9]-*)*[a-z0-9]+\.?)*([a-z0-9]+))(\/[\w?=\.-]*)*$/"
data-validation-regex-message="Enter a valid Web Address"

![screen shot 2015-02-06 at 2 31 14 pm](https://cloud.githubusercontent.com/assets/9268258/6076715/09400812-ae0d-11e4-804c-cba2fe728d63.png)
![screen shot 2015-02-06 at 2 31 32 pm](https://cloud.githubusercontent.com/assets/9268258/6076714/09276e1a-ae0d-11e4-95b8-cd59b72aea6d.png)

**Comment by running-coder** [comment]
Hey ayyaNaga, you are right, the data-validation-regex is currently inverted. I think it adds to confusion having it this way and I will change it in the upcomming version (this weekend). Meanwhile you are able to use the given URL rule.

```javascript
<input type="text"
    size="35"
    maxlength="25"
    class="form-control"
    name="webTab.linkname"
    id="linkname"
    data-validation="[NOTEMPTY, URL]"
    data-validation-message="Enter a valid Web Address">
```
The available rules are given [here](http://www.runningcoder.org/jqueryvalidation/documentation/) with descriptions.

```javascript
var _rules = {
        NOTEMPTY: /./,
        INTEGER: /^\d+$/,
        NUMERIC: /^\d+(?:[,|\s]\d{3})?(?:\.\d+)?$/,
        MIXED: /^[\w\s-]+$/,
        NOSPACE: /\s/,
        TRIM: /^[^\s].*[^\s]$/,
        DATE: /^\d{4}-\d{2}-\d{2}(\s\d{2}:\d{2}(:\d{2})?)?$/,
        EMAIL: /^([^@]+?)@(([a-z0-9]-*)*[a-z0-9]+\.)+([a-z0-9]+)$/i,
        URL: /^(https?:\/\/)?((([a-z0-9]-*)*[a-z0-9]+\.?)*([a-z0-9]+))(\/[\w?=\.-]*)*$/,
        PHONE: /^(\()?\d{3}(\))?(-|\s)?\d{3}(-|\s)\d{4}$/,
        OPTIONAL: /^.*$/,
        COMPARISON: /^\s*([LV])\s*([<>]=?|==|!=)\s*([^<>=!]+?)\s*$/
    }
```

**Comment by running-coder** [comment]
Fixed in 1.5.0
